src/main/java/com/epam/reportportal/utils/properties/PropertiesLoader.java

```diff
@@ -129,16 +130,32 @@ private static Properties loadFromFile() throws IOException {
 	}
 
 	/**
-	 * Reload properties from environment variables.
+	 * Reload properties from system properties.
 	 *
 	 * @param props
 	 * @return props
 	 */
+	public static Properties reloadFromSystemProperties(Properties props) {
+        Properties systemProperties = System.getProperties();
+        for (ListenerProperty listenerProperty : ListenerProperty.values()) {
```

Comment:
This is almost the same cycle like we have in reloadFromEnvVariables. I believe it can be refactored a bit to avoid code duplicating.